cmd/lncli/commands.go

```diff
@@ -2022,13 +2022,16 @@ var updateChannelPolicyCommand = cli.Command{
 		cli.Int64Flag{
 			Name: "time_lock_delta",
 			Usage: "the CLTV delta that will be applied to all " +
-				"forwarded HTLCs",
+				"forwarded HTLCs. If unset, the CLTV delta " +
+				"is left unchanged.",
+			Value: 40,
```

Comment:
Should use `chainreg.DefaultBitcoinTimeLockDelta` here.

Reply:
Also, setting this value here means that whenever the command is used, the time lock delta is set to 40. That's definitely not what we want. What you need is to distinguish between "the user set a value here" (`ctx.IsSet()`) and then send that value on the RPC interface and "the user did not specify a value" and then send 0 on the RPC interface to signal no change. The value of 40 here is only for showing the correct "default: 40" value on the CLI in the help text.

Reply:
I don't think any new calls to `ctx.IsSet()` are needed

`ctx.IsSet()` is used in updateChannelPolicy: https://github.com/lightningnetwork/lnd/blob/master/cmd/lncli/commands.go#L2324

When ctx.IsSet() is false for a flag, the value over RPC is 0 because the variable is initialized to 0.